modules/oe_content_organisation/config/install/core.entity_form_display.node.oe_organisation.default.yml

```
@@ -3,14 +3,16 @@ status: true
 dependencies:
   config:
     - field.field.node.oe_organisation.body
-    - field.field.node.oe_organisation.oe_organisation_overview
+    - field.field.node.oe_organisation.oe_organisation_org_chart
+    - field.field.node.oe_organisation.oe_org_staff_search_link
```

Let's rename this field, if too long let's not sacrifice the prefix, but rather the rest, let's try:

```suggestion
    - field.field.node.oe_organisation.oe_organisation_staff_link
```